src/app/guildmanager/components/guildtable/guildtable.component.ts

```diff
@@ -1,31 +1,28 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { MatTableDataSource } from '@angular/material';
 
-export interface PeriodicElement {
-  name: string;
-  position: number;
-  weight: number;
-  symbol: string;
-}
-
-const ELEMENT_DATA: PeriodicElement[] = [
-  {position: 1, name: 'Hydrogen', weight: 1.0079, symbol: 'H'},
-  {position: 2, name: 'Helium', weight: 4.0026, symbol: 'He'},
-  {position: 3, name: 'Lithium', weight: 6.941, symbol: 'Li'},
-  {position: 4, name: 'Beryllium', weight: 9.0122, symbol: 'Be'},
-  {position: 5, name: 'Boron', weight: 10.811, symbol: 'B'},
-  {position: 6, name: 'Carbon', weight: 12.0107, symbol: 'C'},
-  {position: 7, name: 'Nitrogen', weight: 14.0067, symbol: 'N'},
-  {position: 8, name: 'Oxygen', weight: 15.9994, symbol: 'O'},
-  {position: 9, name: 'Fluorine', weight: 18.9984, symbol: 'F'},
-  {position: 10, name: 'Neon', weight: 20.1797, symbol: 'Ne'},
-];
+import { IGuild } from 'src/app/models/guild';
+import { GuildService } from 'src/app/services/guild.service';
 
 @Component({
   selector: 'app-guild-table',
   templateUrl: './guildtable.component.html',
   styleUrls: ['./guildtable.component.scss']
 })
-export class GuildTableComponent {
-  displayedColumns: string[] = ['position', 'name', 'weight', 'symbol'];
-  dataSource = ELEMENT_DATA;
+export class GuildTableComponent implements OnInit {
+  displayedColumns: string[] = ['dateCreated', 'guildId', 'guildName', 'ownerId'];
+  dataSource: MatTableDataSource<IGuild>;
+  guildz: IGuild[];
```

Comment:
Why guildz?

Reply:
I...don't...know...my inner leet speak was leaking I didn't even realize.  Good catch

Reply:
I can't find this...I think you were somehow looking at an older commit :D